features/steps/project/redirects.rb

```diff
@@ -17,7 +17,7 @@ class Spinach::Features::ProjectRedirects < Spinach::FeatureSteps
   end
 
   step 'I should see project "Community" home page' do
-    Gitlab.config.gitlab.stub(:host).and_return("www.example.com")
+    Gitlab.config.gitlab.should_receive(:host).and_return("www.example.com")
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.